workshop/Lab0/README.md

```diff
@@ -70,7 +52,12 @@ $echo $PATH
 
 3. Convert the binary file to an executable: `chmod +x /usr/local/bin/kubectl`
 
-# Install Docker
-To locally build images and push them to your registry namespace, [install Docker](https://www.docker.com/community-edition#/download). The Docker CLI is used to build apps into images. 
+# Download the Workshop Source Code
+Repo `guestbook` has the application that we'll be deploying. While we're not going to build it we will use the deployment configuration files from that repo. Guestbook application has two versions v1 and v2 which we will use to demonstrate some rollout functionality later. All the configuration files we use are under the directory guestbook/v1.
```

Comment:
can you wrap these lines so we're consistent and it looks like the same person wrote the entire file?